### PR TITLE
index added

### PR DESCRIPTION
**Pull request by Zakarya525**
Please @KamranUmer review my index file, if it has no conflict or improvement needed.. merge to main

### CODE REVIEW

File: index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { StyleSheet, Text, View } from "react-native";
+import React from "react";
+
+const index = () => {
+  return (
+    <View>
+      <Text>index</Text>
+    </View>
```
Comment (KamranUmer) [inline index.jsx:9]: please improve your file structure.

File: index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  );
+};
+
+export default index;
+
+const styles = StyleSheet.create({});
```
Comment (KamranUmer) [inline index.jsx:15]: hi

**Review by KamranUmer — COMMENTED** [review]
I've revieved. please take a look

File: index.jsx (hunk 2 above)
Comment (Zakarya525) [reply index.jsx:15]: youfa

**Review by Zakarya525 — COMMENTED** [review]
please now review again

**Review by KamranUmer — APPROVED** [review]
ok it good now.

**Pull request merged**